hub/core/storage/provider.py

```diff
@@ -8,6 +9,8 @@
 
 
 class StorageProvider(ABC, MutableMapping):
+    autoflush = False
```

Comment:
this and Cachable aren't used anywhere in provider

Reply:
not sure what you mean

Reply:
line 3 and line 12 can be removed, they're useless.